src/smc-webapp/billing.cjsx

```diff
@@ -1852,42 +1853,42 @@ BillingPage = rclass
                         <Panel header={@get_panel_header('credit-card', 'Payment Methods')} eventKey='1'>
                             <PaymentMethods redux={@props.redux} sources={@props.customer.sources} default={@props.customer.default_source} />
                         </Panel>
-                        <Panel header={@get_panel_header('list-alt', 'Subscriptions')} eventKey='2'>
+                        {<Panel header={@get_panel_header('list-alt', 'Subscriptions')} eventKey='2'>
```

Comment:
instead of putting these in brackets, can you just put them in separate render methods?  It makes the code easier to read....